chucks-pick3-web/assets/css/global-style-sheet.css

```diff
@@ -141,3 +141,7 @@ h1 {
     text-align: center;
 
 }
+
+.body-container {
+    z-index: 1;
```

Comment:
Let find out what the z-index is doing for you and then find an alternative solution.

Reply:
Removed z-index and saw it had no functionality to the rest of the web pages after I finished troubleshooting it.